org.jacoco.core.test/src-java7/org/jacoco/core/test/filter/targets/TryWithResources.java

```diff
@@ -109,6 +110,35 @@ private static void nested() {
 		}
 	}
 
+	/**
+	 * In this case bytecode will contain 3 copies of <code>finally</code>
+	 * block, each containing 2 branches, resulting in 6 branches in total. One
+	 * could think that this is artifact of try-with-resources, but the same
+	 * happens without it.
```

Comment:
@marchof just a note: now recalling semantic of `finally` it seems quite logical for me that in case of non-exceptional flow from `catch` there will be one more duplicate of `finally` block and won't be in case of exceptional (`throw`), but IMO this is not obvious at a first glance